torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2111,10 +2111,31 @@ Tensor symeig_backward(const std::vector<torch::autograd::Variable> &grads, cons
 
   auto vh = v.conj().transpose(-2, -1);
 
+  // Based on a little experiment, min_threshold should be
+  //   "max_eigenvalue * matrix_size * machine_eps"
+  // See: https://github.com/pytorch/pytorch/pull/50942
+  // However, as getting max_eigenvalue would need synchronization, I just take
```

Comment:
Should this really be a maximum eigenvalue or a maximum _absolute value_ of eigenvalue?
Maybe `lambda.amax(/*dim=*/-1, /*keepdim=*/true)` (or `lambda.abs().amax(/*dim=*/-1, /*keepdim=*/true)`) is a good option to use here? It should be using a reduction TensorIterator.

Reply:
It's maximum absolute value of eigenvalue. Thanks for the suggestion.
I also changed the formula slightly because it didn't work well with small matrices.